### DISCUSSION TITLE
train data used in test stage

### DISCUSSION THREAD

**Discussion by ruizhang0101** [PaddlePaddle/PaddleSpatial#94] [BaiduKDDCup2022] [answered]
Is it ok to use the train data again to forecast? If yes, could you please inform the path to the train data as the train data is larger than 200MB.

**Comment by ramber1836** [comment, marked as answer]
You don't have to upload the train data. Please refer to the manual (PaddleSpatial/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/README.md) for more details.
